### ISSUE TITLE
[CLI] "use <databaseNameWithTypo>" does not return error 

### ISSUE THREAD

**Issue by bigKS** [influxdb/influxdb#5174]
[ksengal@csdev-ksengal testInflux]$ influx
Visit https://enterprise.influxdata.com to register for updates, InfluxDB server management, and monitoring.
Connected to http://localhost:8086 version 0.9.5.1
InfluxDB shell 0.9.5.1
%> use FAKDBNAME   <~~~~ Typo in name of database... but no error
Using database FAKDBNAME
%> show series
%> show databases
name: databases
---------------
name
FAKEDBNAME

%>

**Comment by pires** [comment]
Currently, there's no validation so no error is expected.

I know where to add this validation but am not sure about the best way to determine if the database exists. @otoolep hints? I thought of:
```go
if _, err := c.ExecuteQuery("SHOW RETENTION POLICIES ON FAKDBNAME"); err != nil {
    // show error to the user
}
```, but if feels very _hackish_.

**Comment by otoolep** [comment]
@pires -- why not simply execute `SHOW DATABASES` and check the list of database names that comes back for the presence of the database?

I also agree that your suggestion would work, but I think the first option is clearer.

In any event, fixing this would be of great value to us!

**Comment by pires** [comment]
@otoolep yeah, it wasn't a good choice, I agree. I should have been clearer about my intent which is simply to validate if the database exists in the _cheapest_ way possible.

Owning this.

**Comment by beckettsean** [comment]
Duplicate of https://github.com/influxdb/influxdb/issues/2758

**Comment by beckettsean** [comment]
https://github.com/influxdb/influxdb/issues/2758#issuecomment-121870741 has good context on possible error messages.

**Comment by pires** [comment]
I believe my PR fixes #2758, as well.
